automation/TaskManager.py

```diff
@@ -366,19 +367,16 @@ def _issue_command(self, browser: Browser,
         start_time = None
         for command_and_timeout in command_sequence.commands_with_timeout:
             command, timeout = command_and_timeout
-            if command[0] in ['GET', 'BROWSE',
-                              'SAVE_SCREENSHOT',
-                              'SCREENSHOT_FULL_PAGE',
-                              'DUMP_PAGE_SOURCE',
-                              'RECURSIVE_DUMP_PAGE_SOURCE']:
+            command.set_visit_crawl_id(browser.curr_visit_id, browser.crawl_id)
+            if isinstance(command, Dump_Flash_Cookies_Command) or isinstance(
+                    command, Dump_Prof_Command):
+                command.set_start_time(start_time)
```

Comment:
Could you please set `start_time` to `time.time()` and remove the local variable.
I know this is a change in behavior but the current behavior was incorrect even though nobody noticed it